docassemble/ALAffidavitOfIndigency/data/questions/affidavit_body.yml

```diff
@@ -725,11 +711,11 @@ attachment:
       - "EAEDC": ${yesno(public_assistance_kinds['EAEDC'])}
       - "VA": ${yesno(public_assistance_kinds['VA Benefits'])}
       - "below_poverty_level": ${yesno(household_income_qualifies) if need_income else "No"}
-      - "weekly": ${weekly}
-      - "biweekly": ${biweekly}
-      - "monthly": ${monthly}
-      - "annually": ${annually}
-      - "income_amount": ${hh_income.total(times_per_year=hh_income.times_per_year)}
+      - "weekly": ${'No'}
+      - "biweekly": ${'No'}
+      - "monthly": ${'Yes'}
+      - "annually": ${'No'}
+      - "income_amount": ${thousands(hh_income.total(times_per_year=12), show_decimals=True)}
```

Comment:
I can see this removes need for intermediate variable, but it does seem slightly clearer for us to show income over the same time period they told us about it, right? can we move the logic in 676 to the attachment block?

Reply:
The issue there is that we have more choices than the form has: the form has weekly, biweekly, monthly, and annually, but we offer more choices (every 3 months and every 6 months I think). So sometimes we will need to standardize the time here. Additionally, the form is mostly consumed by court folks, and the primary purpose is to compare it against the poverty rate, which from what I've seen are given in monthly tables (i.e. https://www.masslegalservices.org/content/federal-poverty-guidelines-2022), though it does seem the official numbers are annual, I'd be willing to do that, even though it doesn't address your original comment.

Reply:
I suggest normalizing anything other than the choices on the form to monthly, as the previous code did. It could be pretty alarming to someone if they entered their current monthly income in a seasonal job and then had it normalized to yearly income in the final PDF. Also prompt some questions from the judge (e.g., you said you earn 30,000/year but now you're saying you only earned 20,000 last year).

OR we could limit our choices to the ones on the form--that just requires users to do math in a few uncommon situations.

Reply:
Gotcha. I think I originally misunderstood the code block, but we should be doing the same thing now.